Travis sometimes failing, sometimes not

When I first created pull request #1573 (documentation changes only), one of the Travis lines failed. I re-ran that specific line and it passed the second time around. Here are the details:

Python 3.6 with RethinkDB backend

Only one of the tests failed. That test was written by @libscott so I will assign this issue to @libscott for further investigation. 

```text
=================================== FAILURES ===================================
________________________ test_elect_disagree_prev_block ________________________
[gw1] linux -- Python 3.6.1 /home/travis/virtualenv/python3.6.1/bin/python
federation_3 = <zip object at 0x7f9d5bf75408>
    @pytest.mark.bdb
    @pytest.mark.genesis
    def test_elect_disagree_prev_block(federation_3):
        [bx, (s0, s1, s2)] = federation_3
        tx = input_single_create(bx[0])
        process_tx(s0)
        process_tx(s1)
>       process_tx(s2)
tests/integration/test_federation.py:143: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/integration/test_federation.py:66: in process_tx
    steps.block_changefeed(timeout=1)
tests/pipelines/stepping.py:135: in <lambda>
    return lambda **kwargs: self.step(name, **kwargs)
tests/pipelines/stepping.py:115: in step
    return task(**kwargs)
tests/pipelines/stepping.py:71: in inner
    r = f(**kwargs)
tests/pipelines/stepping.py:66: in f
    return node.outqueue.get(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
self = <multiprocessing.queues.Queue object at 0x7f9d5a824c88>, block = True
timeout = 0.9999909400939941
    def get(self, block=True, timeout=None):
        if block and timeout is None:
            with self._rlock:
                res = self._recv_bytes()
            self._sem.release()
        else:
            if block:
                deadline = time.time() + timeout
            if not self._rlock.acquire(block, timeout):
                raise Empty
            try:
                if block:
                    timeout = deadline - time.time()
                    if timeout < 0 or not self._poll(timeout):
>                       raise Empty
E                       queue.Empty
/opt/python/3.6.1/lib/python3.6/multiprocessing/queues.py:105: Empty
------------------------------ Captured log setup ------------------------------
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
config_utils.py            231 DEBUG    System already configured, skipping autoconfiguration
connection.py               75 DEBUG    Connection: <class 'bigchaindb.backend.rethinkdb.connection.RethinkDBConnection'>
------------------------------ Captured log call -------------------------------
stepping.py                112 DEBUG    Stepping block_changefeed
stepping.py                112 DEBUG    Stepping block_filter_tx
stepping.py                112 DEBUG    Stepping block_changefeed
stepping.py                112 DEBUG    Stepping block_filter_tx
stepping.py                112 DEBUG    Stepping block_changefeed
============== 1 failed, 483 passed, 14 skipped in 64.33 seconds ===============
```

--- a/tests/integration/test_federation.py
+++ b/tests/integration/test_federation.py
@@ -135,27 +135,6 @@ def test_elect_invalid(federation_3):
 
 @pytest.mark.bdb
 @pytest.mark.genesis
-def test_elect_disagree_prev_block(federation_3):
-    [bx, (s0, s1, s2)] = federation_3
-    tx = input_single_create(bx[0])
-    process_tx(s0)
-    process_tx(s1)
-    process_tx(s2)
-    process_vote(s0, True)
-    for i in range(3):
-        assert bx[i].get_transaction(tx.id, True)[1] == 'undecided'
-    s1.vote.last_voted_id = '5' * 64
-    process_vote(s1, True)
-    for i in range(3):
-        assert bx[i].get_transaction(tx.id, True)[1] == 'undecided'
-    s2.vote.last_voted_id = '6' * 64
-    process_vote(s2, True)
-    for i in range(3):
-        assert bx[i].get_transaction(tx.id, True)[1] is None
-
-
-@pytest.mark.bdb
-@pytest.mark.genesis
 def test_elect_sybill(federation_3):
     [bx, (s0, s1, s2)] = federation_3
     tx = input_single_create(bx[0])

